### DISCUSSION TITLE
SMB support...

### DISCUSSION THREAD

**Discussion by 66mkiqdv** [deckerst/aves#1091] [Ideas]
This app is great for locally stored photos but what are the plans for photos stored on a NAS? Is this in the pipeline to be implemented?

**Comment by deckerst** [comment]
There are already issues related to this topic:
- #126
- #709
- #857

Feel free to upvote/subscribe to these if you're interested.

More generally about plans, I'm limited by time and energy I can put into this app. And my intentions are found within the priority labels I assign. You'll notice the issues above are all `P3`. In other words I'd like to do it eventually but it requires thought and lots of energy.
